static/templates/suggestion.json

```diff
@@ -1,7 +1,7 @@
 [
   {{#suggestions}}{{^first}},{{/first}}
   {
-    "value" : "{{value}}",
+    "value" : "{{url}}",
```

Comment:
`url` will be HTML escaped/encoded here (in particular `&` will be replaced with `&amp;`). That may seem wrong, but the current design of the full (front-end + back-end) system [assumes](https://github.com/kiwix/kiwix-lib/issues/398#issuecomment-789169478) that `value` is HTML-decoded in the front-end. Please put a comment here making that explicit.

Reply:
what you mean here is I should write `&amp;` in place of `&` in L430 InternalServer.cpp, other than that what comment should be added?
something like `url received should be html encoded`

Reply:
See http://mustache.github.io/mustache.5.html. The `{{variable}}` syntax applies HTML escaping. For a URL that may seem wrong. The comment must say that this is intentional, since the receiving side in taskbar.js un-escapes this field.